Assets/Mirror/Core/NetworkBehaviour.cs

```diff
@@ -9,12 +9,25 @@ namespace Mirror
     // SyncMode decides if a component is synced to all observers, or only owner
     public enum SyncMode { Observers, Owner }
 
+    // SyncDirection decides if a component is synced from:
+    //   * server to all clients
+    //   * owner client, to server, to all other clients
+    //
+    // naming: 'CLIENT_TO_SERVER' etc. instead of 'ClientAuthority', because
+    // that wouldn't be accurate. server's OnDeserialize can still validate
+    // client data before applying. it's really about direction, not authority.
+    public enum SyncDirection { SERVER_TO_CLIENT, CLIENT_TO_SERVER }
```

Comment:
Code style wise `ServerToClient` and `ClientToServer` would be more in line with the rest of the project, any reason for the different style?

Reply:
done